### ISSUE TITLE
Base Plugins

### ISSUE THREAD

**Issue by buddytex** [kennyvv/SharpMC#5]
Should there be a few basic, pre-packaged plugins that provided standard commands? (op, gamemode, tps, etc.) If so it might be a good idea to branch those to another repository to keep things clean.

**Comment by kennyvv** [comment]
I agree on that, i don't think they should be built in to do the server, i guess i could make this like a 'Core Commands Plugin' or something. We'll see! :+1:

**Comment by kennyvv** [comment]
Maybe a list of Commands would be a good idea:
I guess most of the commands here http://minecraft.gamepedia.com/Commands#Summary_of_commands would be a good thing to add.

Maybe some others?

**Comment by kennyvv** [comment]
The plugin that will add these: https://github.com/SharpMC/SharpCore

**Comment by djlw78** [comment]
@buddytex @kennyvv O wow! This really got done! No seriously if you guys are still remotely there I'm going to try to update this and aim for an alternative to bukkit/spigot/.../etc

**Comment by kennyvv** [comment]
@djlw78 Interesting seeing you would even want to... This project hasn't been updated in a long time with a reason: it was (and still is) horribly written. Feel free to contact me on discord tho, my discord tag is #7750

**Comment by Nev3rGiveUp** [comment]
The code can always be improved. The idea is amazing. I feel bad that I'm still learning C# and can't help with it. It's really good project that somebody should investigate in. Learning Java just to write plugins for Spigot is not worth it. C# the best :)

**Comment by HurricanKai** [comment]
Im Currently Trying to rework this on my own. But i woud love to work together with someone else!
I already Got an version running. ATM im trying to update to Use Substrate and with that 1.12.
@kennyvv i wanted to ask from where you got the Package Send, Receive "id"s if you can remember?

EDIT: nvm just found it on wiki.vg xD

**Comment by linuxcodefire** [comment]
@HurricanKai I would like to help you. Give me a day to pull things together but I'll assist. I am pretty intermediate in c#

**Comment by linuxcodefire** [comment]
(@djlw78) alt account

**Comment by djlw78** [comment]
Confirmed.

**Comment by HurricanKai** [comment]
@djlw78 @linuxcodefire nice!

If youve got discord maybe wie coud write there? This issue ist the best place for that i think :D

**Comment by HurricanKai** [comment]
I guess @djlw78 / @linuxcodefire are no longer interested? One year and a day later I created a mc server not once, but twice, well the second one im working on...
